fsi/fsi.go

```diff
@@ -152,6 +155,10 @@ func (fsi *FSI) Unlink(dirPath, refStr string) error {
 		return err
 	}
 
+	// attempt to remove the directory, this will error if the directory is
+	// not empty. We intentionally ignore this error
+	os.Remove(dirPath)
```

Comment:
Suggestion: be explicit about which errors that we ignore, like "directory not empty", but still acknowledge other errors such as "permission denied".